.travis.yml

```diff
@@ -1,15 +1,30 @@
 language: node_js
 
+sudo: false
+
+os:
+  - osx
+  - linux
+
 node_js:
   - 'stable'
 
+before_install:
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
+      sh -e /etc/init.d/xvfb start;
+      sleep 3;
+    fi
+
 install:
   - npm install
+  - npm run vscode:prepublish
```

Comment:
This does the same as `tsc -p ./` on line 24. (That being said, I would prefer you overwrite that with this `npm run` version)

Reply:
@EricJizbaMSFT You mean this?

install:
  - npm install

script:
  - npm run vscode:prepublish
  - npm test